bls/src/main/java/org/dashj/bls/AugSchemeMPL.java

```diff
@@ -0,0 +1,67 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 4.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+package org.dashj.bls;
+
+public class AugSchemeMPL extends CoreMPL {
```

Comment:
This is the first of many Java wrapper classes that wrap every method (except those that are explicitly skipped) of a particular C++ class.